.circleci/src/commands/@docker-commands.yml

```diff
@@ -0,0 +1,132 @@
+docker-login:
+  steps:
+    - run:
+        name: Docker login
+        command: echo "$DOCKERHUB_PASS" | docker login -u "$DOCKERHUB_USERNAME" --password-stdin
+
+docker-logout:
+  steps:
+    - run:
+        when: always
+        name: Log out of Docker
+        command: docker logout
+
+docker-tag-images:
+  description: 'Update tags of docker images'
+  parameters:
+    tag:
+      type: enum
+      enum: ['prerelease', 'edge', 'current', '__version__']
+    service:
+      type: enum
+      enum: ['discovery-provider', 'creator-node', 'identity-service', 'ddex', 'all']
+  steps:
+    - run:
+        name: Get crane
+        command: |
+          VERSION=v0.19.1
+          OS=Linux
+          ARCH=x86_64
+          curl -sL "https://github.com/google/go-containerregistry/releases/download/${VERSION}/go-containerregistry_${OS}_${ARCH}.tar.gz" > go-containerregistry.tar.gz
+          tar -zxvf go-containerregistry.tar.gz -C $HOME/bin/ crane
+    - docker-login
+    - run:
+        name: Update docker containers with release tags
+        command: |
+          discovery=(
+            comms
+            dashboard
```

Comment:
one callout: iirc running `audius-compose push dashboard` actually builds 2 targets like `dashboard-stage` and `dashboard-prod` unless that change is gone